test/parallel/test-http2-compat-serverresponse-end.js

```diff
@@ -25,16 +25,22 @@ const {
   // but callback will only be called once
   const server = createServer(mustCall((request, response) => {
     response.end('end', 'utf8', mustCall(() => {
-      response.end(mustNotCall());
+      response.end(mustCall());
       process.nextTick(() => {
-        response.end(mustNotCall());
+        response.end(mustCall(err => {
```

Comment:
You can use `common.expectError()` in here instead of using `common.mustCall()`.